documentation/docs/core-concepts/anatomy-of-a-request.md

```diff
@@ -57,11 +57,11 @@ In the above example, because `SecondPlugin` depends on `FirstPlugin`, the `cons
 2. `StandalonePlugin`
```

Comment:
wouldn't `StandalonePlugin` run last since it's registered last?

Reply:
lol, yeah. I'll fix.